### DISCUSSION TITLE
Creating a wrapper to use `next.js` together with `expo`

### DISCUSSION THREAD

**Discussion by MrOxMasTer** [expo/expo#33281] [Proposals and Ideas]
I know that `expo` has a wrapper that allows expo to be used in conjunction with `next.js`:
https://docs.expo.dev/guides/using-nextjs/
But `page` routing is no longer relevant, and `app router` is not experimental. 
I think it would be a revolution in the js world if such 2 js well done frameworks come together. It would be great to create such a wrapper.
Single routing + pwa + mobile app. That would compile on web / ios / android as well.
I know expo has an option to use `web`, but I think there are better ones in each area: expo for mobile app, and next.js for web.
It would be nice to write once, and execute everywhere. At least the models that expo and next.js have are very similar in many ways: routing, and more recently the server side has emerged, which would make for a convenient wrapper to create a single app for all platforms.
At that - with the option to choose server `next.js` or `expo` usage

**Comment by edward-meister** [comment]
Agreed, much needed
